sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -630,11 +630,11 @@ class LinkedEntity(DictMixin):
     :ivar data_source: Data source used to extract entity linking,
         such as Wiki/Bing etc.
     :vartype data_source: str
-    :ivar str bing_id: Bing unique identifier of the recognized entity. Use in conjunction
+    :ivar str bing_entity_search_api_id: Bing unique identifier of the recognized entity. Use in conjunction
```

Comment:
late to the party, but consider modifying the docstirng to `Bing Entity Search unique identifier ...`